src/main/kotlin/cqrs/core/queries/ProductQueries.kt

```diff
@@ -0,0 +1,5 @@
+package cqrs.core.queries
+
+data class GetProductByIdQuery(val id: String)
+data class GetAllProductsQuery(val placeholder: Boolean = true)
```

Comment:
not sure what the Boolean does yet, but if there's a default value, then it could also just be a boolean (with small b) as it can never be null, no?